SPECS/python3/python3.spec

```diff
@@ -155,7 +157,17 @@ Provides:       python%{majmin_nodots}-test = %{version}-%{release}
 The test package contains all regression tests for Python as well as the modules test.support and test.regrtest. test.support is used to enhance your tests while test.regrtest drives the testing suite.
 
 %prep
-%autosetup -p1 -n Python-%{version}
+# We need to patch setuptools later, so manually manage patches with -N
+%autosetup -p1 -n Python-%{version} -N
+
+# Ideally we would use '%%autopatch -p1 -M 999', but unfortunately the GitHub CI pipelines use a very old version of rpm which doesn't support it.
```

Comment:
I'm curious, if it's possible to update the pipeline to use Ubuntu 22.04. This used to resolve my issues with some other PR checks. Unless the check that's failing for you is already there. In this case, I guess we're out of luck.;)

Reply:
Looks like that worked, good idea!

Reply:
Ah, unfortunately I hadn't considered that python3 is part of the toolchain, which needs to parse its SRPM using the host tools (ie old rpm). I'll need to run with the multiple %patch version